fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/data/GenericResultsetData.java

```diff
@@ -27,10 +27,15 @@ public final class GenericResultsetData {
 
     private final List<ResultsetColumnHeaderData> columnHeaders;
     private final List<ResultsetRowData> data;
+    private Integer totalReportSize;
```

Comment:
... the class is called "GenericResultsetData"... is it then really necessary to have the word "report" in this variable? We could name things like in Spring Data... just a suggestion.

Reply:
changed to totalItem